<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayQueueView</class>
 <widget class="QWidget" name="PlayQueueView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>61</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>61</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PlayQueueWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#PlayerQueueView {
	background-color: #333350;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="topContainer">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="queueContainer">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="queueFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#queueFrame {
	background-color: #000;
	border-top: 1px solid #26253c;
	border-right: 1px solid #6d6d7f;
	border-bottom: 1px solid #6d6d7f;
	border-left: 1px solid #26253c;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
       <widget class="QTreeView" name="queueList">
        <property name="font">
         <font>
          <family>Bitstream Vera Sans Mono</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTreeView {
	color: #00e800;
	background-color: #000000;
	border-top: 3px solid #26253c;
	border-right: 3px solid #6d6d7f;
	border-bottom: 3px solid #6d6d7f;
	border-left: 3px solid #26253c;
    alternate-background-color: #10101a;
}

QTreeView::item {
	height: 45px;
	padding-right: 16px;
}

QTreeView::item:selected {
	color: #000;
    background-color: #00e800;
}

QTreeView::item:selected:!active {
	color: #000;
	background-color: #00e800;
}

QTreeView::item:selected:active {
	color: #000;
    background-color: #00e800;
}

QScrollBar:vertical {
	background-color: #191926;
	width: 0px;
}

QScrollBar::handle:vertical {
	background-color: #b0995e;
}

 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	height: 0px;
	width: 0px;
	background-color: transparent;
	background: none;
	border: none;
}

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
	height: 0px;
	width: 0px;
	border: none;
 }

 QScrollBar::add-line:vertical {
    background: none;
	height: 0px;
	width: 0px;
	border: none;
 }

 QScrollBar::sub-line:vertical {
    background: none;
	height: 0px;
	width: 0px;
	border: none;
 }


QScrollBar:horizontal {
	background-color: #191926;
	height: 4px;
}

QScrollBar::handle:horizontal {
	background-color: #b0995e;
}

 QScrollBar::left-arrow:horizontal, QScrollBar::right-arrow:horizontal {
	height: 0px;
	width: 0px;
	background-color: transparent;
	background: none;
	border: none;
}

 QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
	background: none;
	height: 0px;
	width: 0px;
	border: none;
 }

 QScrollBar::add-line:horizontal {
    background: none;
	height: 0px;
	width: 0px;
	border: none;
 }

 QScrollBar::sub-line:horizontal {
    background: none;
	height: 0px;
	width: 0px;
	border: none;
 }

QHeaderView::section {
    background-color: #bccdd5;
    color: #333350;
    border: none;
	height: 28px;
	font-size: 12pt;
	font-weight: bold;
	padding-right: 8px;
}</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
       </widget>
        
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="scrollBarContainer">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QSlider" name="scrollBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>8</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider::handle:vertical {
	width: 8px;
	height: 18px;
	background-color: transparent;
	border-image: url(:assets/posHandle_vert.png);
	background: none;
	background-repeat: none;
}

QSlider::handle:vertical:pressed {
	border-image: url(:assets/posHandle_vert_p.png);
}

QSlider::groove:vertical {
	background: transparent;
}

#scrollBar {
	background-color: #2c2b43;
	border-top: 2px solid #26253c;
	border-right: 1px solid #6d6d7f;
	border-bottom: 1px solid #6d6d7f;
	border-left: 2px solid #26253c;
	padding-top: -2px;
	padding-right: -1px;
	padding-bottom: -1px;
	padding-left: -2px;
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
